vm/src/obj/objstr.rs

```diff
@@ -518,6 +518,18 @@ impl PyString {
         do_cformat(vm, format_string, values.clone())
     }
 
+    #[pymethod(name = "__rmod__")]
+    fn rmod(&self, values: PyObjectRef, vm: &VirtualMachine) -> PyResult {
+        let format_string_text = &self.value;
+        let _format_string = CFormatString::from_str(format_string_text)
+            .map_err(|err| vm.new_value_error(format!("{}", err)))?;
```

Comment:
```suggestion
            .map_err(|err| vm.new_value_error(err.to_string()))?;
```

Reply:
That code found on str.__mod__ too. I suggest that code should be moved to another PR.

Reply:
Sounds good, thanks for contributing!